### PR TITLE
Add Theme Editor documentation

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: docs/End-User-Guide.md [hunk 1]
```diff
@@ -309,6 +315,20 @@ Click the **Device** icon and select the desired device from the drop-down menu.
...
+<img src="images/theme-editor.png" alt="Survey Creator - The Themes tab" width="100%">
+
+### How to: Save a Custom Theme
+
+SurveyJS themes are JSON objects with CSS variables and other theme settings. Once you finish customization, click the Export button to download a JSON object with your custom theme.
+
+<img src="images/theme-export.png" alt="SurveyJS Theme Export">
+
+Other capabilities depend on features implemented by your development team. For instance, the following help topic describes how developers can save themes on a server and load them from it: [Save and Load Custom Themes](/survey-creator/documentation/theme-editor#save-and-load-custom-themes).
+
```
Comment (JaneSjs) [inline docs/End-User-Guide.md:331]: ### How to: Load an Existing Theme

You can also load a theme from a file. Click the Import button to open a file which contains a custom theme.

<img src="..." alt="SurveyJS Theme Import>

File: docs/theme-editor.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  request.open('POST', url);
+  request.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+  request.addEventListener('load', () => {
+      callback(saveNo, true);
+  });
+  request.addEventListener('error', () => {
+      callback(saveNo, false);
+  });
+  request.send(JSON.stringify(json));
+}
```
Comment (JaneSjs) [inline docs/theme-editor.md:78]: Fetch API?

```
function saveThemeJson(url, json, saveNo, callback) {
  fetch(url, {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json;charset=UTF-8'
    },
    body: JSON.stringify(json)
  })
  .then(response => {
    if (response.ok) {
      callback(saveNo, true);
    } else {
      callback(saveNo, false);
    }
  })
  .catch(error => {
    callback(saveNo, false);
  });
}
```

File: docs/theme-editor.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+const creator = new SurveyCreatorModel(creatorOptions);
+```
+
+## Apply a Custom Theme
+
+Theme Editor produces a JSON object with CSS variables and other theme settings. For information on how to obtain this object and apply it to a survey, refer to the following help topic in Form Library documentation: [Create a Custom Theme](/form-library/documentation/manage-default-themes-and-styles#create-a-custom-theme).
+
+## Save and Load Custom Themes
+
+Theme JSON objects can be persisted on your server to let users save, share, and restore custom themes or switch between them. To save a theme JSON object, implement the `saveThemeFunc` function. It accepts two arguments:
```
Comment (annnke) [inline docs/theme-editor.md:33]:
```suggestion
Theme JSON objects can be stored on your server to let users save, share, and restore custom themes or switch between them. To save a theme JSON object, implement the `saveThemeFunc` function. It accepts two arguments:
```

File: docs/theme-editor.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      callback(saveNo, false);
+    }
+  })
+  .catch(error => {
+    callback(saveNo, false);
+  });
+}
+```
+
+To load a theme JSON object into Theme Editor, assign the object to Survey Creator's [`theme`](https://surveyjs.io/survey-creator/documentation/api-reference/survey-creator#theme) property. The following code takes a theme JSON object from the `localStorage`:
```
Comment (annnke) [inline docs/theme-editor.md:88]:
```suggestion
To load a theme JSON object into Theme Editor, assign the object to Survey Creator's [`theme`](https://surveyjs.io/survey-creator/documentation/api-reference/survey-creator#theme) property. The following code takes a theme JSON object from `localStorage`:
```

**Review by annnke — APPROVED** [review]

**Pull request merged**